src/python/pants/backend/python/tasks/pytest_run.py

```diff
@@ -111,7 +110,12 @@ def register_options(cls, register):
                   "it's best to use an absolute path to make it easy to find the subprocess "
                   "profiles later.")
 
-    register('--options', type=list, fingerprint=True, help='Pass these options to pytest.')
+    register('--options', type=list, fingerprint=True,
+             removal_version='1.7.0.dev0',
+             removal_hint='You can supply py.test options using the generic pass through the args '
```

Comment:
s/pass through the args/pass through args/

Reply:
Fixed